dev/io.openliberty.netty.internal/src/io/openliberty/netty/internal/NettyFramework.java

```diff
@@ -13,6 +13,7 @@
 import java.util.Set;
 import java.util.concurrent.Callable;
 import java.util.concurrent.FutureTask;
+import java.util.concurrent.atomic.AtomicBoolean;
```

Comment:
Is this needed here?  I don't see it referenced.

Reply:
Sorry I missed this, I think it's a leftover reference. I'll remove this